### ISSUE TITLE
Rest Api audio no player

### ISSUE THREAD

**Issue by josep112** [chatwoot/chatwoot#6027]
Describe the bug

When sending an mp3 or ogg or m4a audio through the api, the player does not show only a file to download

to reproduce

curl -X POST http://localhost:3000/api/v1/accounts/1/conversations/8/messages --header 'Content-Type: multipart/form-data; boundary=----WebKitFormBoundary' --header 'api_access_token: Ndsi3RiuHsamRWyxKJCsj34e' --form 'attachments[]=@audio.mp3' --form message_type=incoming --form file_type=audio

I also tried --form file_type=audio/mpeg and --form file_type=audio/ogg

Share a clear and concise description of what you expected to happen.

<img width="1512" alt="Captura de Tela 2022-12-06 às 10 33 40" src="https://user-images.githubusercontent.com/12038013/205927698-1d8b17e2-a40e-43e0-935d-53a37a5bdf34.png">

  logs

WHAT I COULD VERIFY IS THAT IT ALWAYS COMES AS
file_type":"file"

{"id":302,"content":null,"inbox_id":1,"conversation_id":8,"message_type":0,"content_type":"text","content_attributes":{},"created_at": 1670333383,"private":false,"source_id":null,"sender":{"additional_attributes":{},"custom_attributes":{},"email":null,"id":9,"identifier":null ,"extension":null,"data_url":"https://chatbots.chatbots.net.br/rails/active_storage/blobs/redirect/eyJfcmFp bHMiOnsibWVzc2FnZSI6IkJBaHBadz09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--a99eef4817153a81b15cb2fe7c8051fa5cf89b69/audio.mp3","thumb_url":"","file_size"}]:17709

Environment

  self-hosted

Desktop (please complete the following information) (If applicable)

TESTED IN ALL BROWSERS chrome, firefox, safari

**Comment by giquieu** [comment]
@iamsivin can i check this?

**Comment by josep112** [comment]
Hi, I sent an audio file in mp3 format via the website's web widget and it arrives with the player in the agent, the problem seems to be just sending the api, any news about this case, thank you in advance

**Comment by iamsivin** [comment]
@josep112 We have tested it, and it's working fine. You can see this in the Loom video attached to this.
https://www.loom.com/share/4c7e5ca56d0440019b88678570e5fae4

**Comment by josep112** [comment]
https://user-images.githubusercontent.com/12038013/207309753-06113f25-79fb-4530-98ed-a9fb4b09adee.mov

See for me it doesn't work, I'm using it hosted the version is v2.11.0 my curl link is below what could be happening?



curl --location -g --request POST 'http://localhost:3000/api/v1/accounts/1/conversations/67/messages' \
	--header 'Content-Type: multipart/form-data; boundary=----WebKitFormBoundary' \
	--header 'api_access_token: Ndsi3RiuHsamRWyxKJCsj34e' \
	--form 'attachments[]=@"/whatsapp/audio.mp3"' \
	--form 'content="audio"' \
	--form 'message_type="incoming"' \
	--form 'file_type="audio"'
	
	look at the answer

{
	"id": 1129,
	"content": "audio",
	"inbox_id": 1,
	"conversation_id": 67,
	"message_type": 0,
	"content_type": "text",
	"content_attributes": {},
	"created_at": 1670931954,
	"private": false,
	"source_id": null,
	"sender": {
		"additional_attributes": {},
		"custom_attributes": {},
		"email": null,
		"id": 62,
		"identifier": null,
		"name": "Chatwoot",
		"phone_number": "+55XXXXXXXX",
		"thumbnail": "http://localhost:3000/rails/active_storage/representations/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBYkU9IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--0e550e4a50694b97d18b68cc7d8c4c9d03191518/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdCem9MWm05eWJXRjBTU0lJYW5CbkJqb0dSVlE2QzNKbGMybDZaVWtpRERJMU1IZ3lOVEFHT3daVSIsImV4cCI6bnVsbCwicHVyIjoidmFyaWF0aW9uIn19--61db38bceee59871131e6df98e5b65fcfd070215/56105953_391646908341536_9067922726437519360_n.jpg",
		"type": "contact"
	},
	"attachments": [{
		"id": 142,
		"message_id": 1129,
		"file_type": "file",
		"account_id": 1,
		"extension": null,
		"data_url": "http://localhost:3000/rails/active_storage/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBYjg9IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--c03175ee79f192f847a5e958631628b930b2ba3b/audio.mp3",
		"thumb_url": "",
		"file_size": 387117
	}]
}

**Comment by iamsivin** [comment]
@giquieu We have tested it again using the CURL command, and working fine. Can you share the file with us that you used here for testing?

**Comment by josep112** [comment]
Yes of course, that same audio works perfectly on the web, follow it
https://186.232.111.36/ChatBots-FTP-1801161920304080baixar-arquivos/audio.mp3

**Comment by josep112** [comment]
@iamsivin See what I discovered:
If I send this audio through my computer via curl it comes with the player, if I send the same audio but inside the chatwoot server it comes as a file, any light?

**Comment by josep112** [comment]
https://user-images.githubusercontent.com/12038013/207590300-364bf82b-5068-4503-8929-66ec1235a9c0.mov

**Comment by muhsin-k** [comment]
@josep112 Thanks for reporting the issue.  We could be able to reproduce this issue from our side.

The attachment content type in the CURL command is saved as  `application/octet-stream`.

![Screenshot 2022-12-15 at 4 22 15 PM](https://user-images.githubusercontent.com/12408980/207851784-df00a9ee-dbca-4c90-9b6e-dd41e1b155d2.png)


But via postman, it is saved as `audio` format.

![Screenshot 2022-12-15 at 4 21 45 PM](https://user-images.githubusercontent.com/12408980/207851764-1dd19a29-ab8b-4877-bb39-83e64a6ea718.png)

We will work on this fix in the upcoming releases.
